custom/icds_reports/ucr/data_sources/dashboard/primary_private_school_form_ucr.json

```diff
@@ -0,0 +1,240 @@
+{
+  "domains": [
+    "icds-dashboard-qa",
+    "sankalp_cas",
+    "zohaib-sandbox",
+    "akshita-sandbox",
+    "sunaina-sandbox",
+    "laurence-project-1",
+    "jessica-icds-cas",
+    "marissa-test",
+    "derek-icds-sandbox",
+    "priyanka-app",
+    "shrena-dev",
+    "aparatest",
+    "reach-sandbox",
+    "reach-dashboard-qa",
+    "reach-test",
+    "icds-cas",
+    "cas-lab"
+  ],
+  "server_environment": [
+    "india",
+    "icds"
+  ],
+  "config": {
+    "table_id": "static-dashboard_primary_private_school",
+    "display_name": "Forms - Primary private school (Static) (Dashboard)",
+    "referenced_doc_type": "XFormInstance",
+    "description": "Primary private form used by dashboard to exclude children from pse eligible",
+    "base_item_expression": {
+      "type": "coalesce",
+      "expression": {
+        "type": "property_path",
+        "property_path": ["form", "case_load_person_0", "case", "@case_id"]
+      },
+      "default_expression": {
+        "type": "constant",
+        "constant": [0]
+      }
+    },
```

Comment:
Why does this need a base_item_expression? from the example XML, there's only one case that could be in the form and it doesn't appear that this data source ever references the iteration number. I think that this and the `repeat_iteration` in the primary key could be dropped

Reply:
Yeah, that makes sense, and that's why repeat_iteration will be always 0. CLeaned the UCR!